src/Ookii.Dialogs.WinForms/InputDialogForm.cs

```diff
@@ -67,6 +67,36 @@ public bool UsePasswordMasking
             set { _inputTextBox.UseSystemPasswordChar = value; }
         }
 
+        public bool MultiLine
+        {
+            get { return _inputTextBox.Multiline; }
+            set {
+                if (value != _inputTextBox.Multiline)
+                {
+                    _inputTextBox.Multiline = value;
+
+                    if(value == true)
```

Comment:
Here you should do `if (value)` instead of `if(value == true)` because `value` is already a `bool` which is what the `if` is expecting.